specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -1913,6 +1913,84 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/containerlogs": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Gets the last lines of docker logs for the given site",
+        "description": "Gets the last lines of docker logs for the given site",
+        "operationId": "WebApps_GetWebSiteContainerLogs",
```

Comment:
And maybe here too?

``` json
 "produces": [
     "application/zip"
  ],
```

(and for the other APIs too...)

Reply:
Updated

Reply:
Same for this one - get `application/json` for content type, which probably requires a server-side fix and the response should be `byte`.